Exiled.API/Features/Player.cs

```diff
@@ -1276,6 +1276,8 @@ public void Handcuff(Player cuffer)
             if (!IsCuffed && Vector3.Distance(Position, cuffer.Position) <= 130f)
             {
                 Cuffer = cuffer;
+                Inventory.SetDisarmedStatus(cuffer.Inventory);
```

Comment:
The `Cuffer` property setter already calls `Inventory.SetDisarmedStatus()`, so this is redundant.